Formula/v8.rb

```diff
@@ -15,98 +16,213 @@ class V8 < Formula
     sha256 "7bcd1bbd66c11305eeea0c36ca472de8a639f511abe0909c8815b1208dbce7b6" => :mavericks
   end
 
-  option "with-readline", "Use readline instead of libedit"
+  # https://bugs.chromium.org/p/chromium/issues/detail?id=620127
+  depends_on :macos => :el_capitan
 
-  # not building on Snow Leopard:
-  # https://github.com/Homebrew/homebrew/issues/21426
-  depends_on :macos => :lion
-
-  # gyp doesn't run under 2.6 or lower
+  # depot_tools/GN require Python 2.7+
   depends_on "python@2" => :build
 
-  depends_on "readline" => :optional
-
   needs :cxx11
 
-  # Update from "DEPS" file in tarball.
-  # Note that we don't require the "test" DEPS because we don't run the tests.
-  resource "gyp" do
-    url "https://chromium.googlesource.com/external/gyp.git",
-        :revision => "4ec6c4e3a94bd04a6da2858163d40b2429b8aad1"
+  resource "depot_tools" do
+    url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
   end
 
-  resource "icu" do
-    url "https://chromium.googlesource.com/chromium/deps/icu.git",
-        :revision => "c291cde264469b20ca969ce8832088acb21e0c48"
+  resource "chromium-buildtools" do
```

Comment:
Are these listed somewhere upstream as to which commit these resources should be pegged to or are we just using the tip of each branch at the time of update?

Reply:
The `DEPS` files in the various repos may or may not include dependencies on other things. Currently, V8 itself has [these deps](https://github.com/v8/v8/blob/6.6.346.26/DEPS#L13-L66), and the buildtools dependency of V8 has [these deps](https://chromium.googlesource.com/chromium/buildtools.git/+/2888931260f2a32bc583f005bd807a561b2fa6af/DEPS).

Reply:
Alright. Let's retain a note somewhere near the top of the resources of where we're pulling the commits from, so it's easier for others to update in future.